internship2018-1/src/products.html

```diff
@@ -0,0 +1,128 @@
+        <script src="./assets/bundle.js"></script>
+    </head>
+    <body>
+        <!-- Navigation start -->
+        <div class="container">
+            <nav class="navbar navbar-expand-lg navbar-light bg-light">
+                <a class="navbar-brand" href="./index.html">
+                    <img src="/assets/image.png" width="30" height="30" alt="">
+                    Softheme
+                </a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                    <div class="navbar-nav">
+                    <a class="nav-item nav-link" href="./index.html">Home</a>
+                    <a class="nav-item nav-link" href="./lesson3.html">Lesson 3</a>
+                    <a class="nav-item nav-link active" href="./products.html">Products</a>
+                    </div>
+                </div>
+            </nav>           
+        </div>
+        <!-- Navigation end -->
+        <center><h1>Меню</h1>
+        <h4>Суші роли на будь-який смак</h4></center>
+        <h1>Products</h1>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-4">
+                    <h4 align="center">Суші Токіо</h4>
+                    <img class="image" src="assets/products/1.jpeg" width="100%" /><br/>
+                    <div class="clearfix">
+                        <div class="pull-left"><h4>70 uah</h4></div>
+                        <div class="pull-right"><button class="btn btn-default btn-lg">order</button></div>
+                    </div>
+                </div>
+                <div class="col-md-4">
+                    <h4 align="center">Суші Окуда</h4>
+                    <img class="image" src="assets/products/2.jpeg" width="100%" /><br/>
+                    <div class="clearfix">
+                        <div class="pull-left"><h4>60 uah</h4></div>
+                        <div class="pull-right"><button class="btn btn-default btn-lg">order</button></div>
+                    </div>
+                </div>
+                <div class="col-md-4">
+                    <h4 align="center">Суші Каппа</h4>
+                    <img class="image" src="assets/products/3.jpeg" width="100%" /><br/>
+                    <div class="clearfix">
+                        <div class="pull-left"><h4>55 uah</h4></div>
+                        <div class="pull-right"><button class="btn btn-default btn-lg">order</button></div>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
```

Comment:
You don't need class="row" for each 3 products. Omit it and next products with be moved to next line automatically